Resources/public/templates/fields/edit/selection.hbt

```diff
@@ -9,6 +8,11 @@
         </label>
     </div>
     <div class="pure-u ez-editfield-input-area ez-default-error-style">
+        <div class="ez-fielddescription-tooltip ez-tail-down-tooltip">
```

Comment:
`{{#if fieldDefinition.descriptions.[eng-GB] }}|` is missing but actually, instead of repeating the same code again and again, why don't you use a partial like it was done before ?